dask/dataframe/tests/test_dataframe.py

```diff
@@ -172,6 +172,20 @@ def test_Index():
         pytest.raises(AttributeError, lambda: ddf.index.index)
 
 
+def test_Axes():
```

Comment:
nit but can we lower case these?
```suggestion
def test_axes():
```

Reply:
Absolutely. Adjusted in https://github.com/dask/dask/pull/8069/commits/876b8ea86f3ec4283e61f25213c145cbdb6e3a65